repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -366,6 +369,25 @@ const createProductDetails = async (
         productDetailsElements.push({ id: 'time-restriction', name: 'Only valid during term time', content: ['Yes'] });
     }
 
+    const hasCaps = (await getCaps(noc)).length > 0;
+    const isDevOrTest = process.env.NODE_ENV === 'development' || process.env.STAGE === 'test';
```

Comment:
This cannot be called here - it must be done in the serverSideProps and passed through as a prop